static/src/javascripts/projects/common/modules/audio/AuralAid.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+import { React, styled } from '@guardian/dotcom-rendering/packages/guui';
+
+const VisuallyHidden = styled('span')({
+    height: '1px',
+    overflow: 'hidden',
+    position: 'absolute',
+    top: '-10px',
+    width: '1px',
+});
+
+type Props = {
+    text: string,
+};
+
+export default function({ text }: Props) {
+    return <VisuallyHidden>{text}</VisuallyHidden>;
+}
```

Comment:
Not a change request, but just so you know but we have a screen reader only style "mixin" available in our `pasteup` library. eg.

`import { screenReaderOnly } from '@guardian/dotcom-rendering/packages/mixins'`

`const VisuallyHidden = styled('span')(screenReaderOnly);`

Reply:
Thanks, will incorporate if we go ahead with this component